tensorflow/python/ops/sparse_ops.py

```diff
@@ -2675,13 +2675,21 @@ def sparse_softmax(sp_input, name=None):
   Example:
 
   ```python
-  import tensorflow as tf
-  shape = [2, 2, 2]  # 3-D SparseTensor
-  values = np.asarray([[[0., np.e], [1., 0.]], [[np.e, 0.], [np.e, np.e]]])
-  indices = np.vstack(np.where(values)).astype(np.int64).T
-  values = values[np.where(values)] # Flatten values
-  result = tf.sparse.softmax(tf.sparse.SparseTensor(indices, values, shape)) 
-  print(tf.sparse.to_dense(result))
+  >>> import tensorflow as tf
```

Comment:
You don't need this line.

Also, you don't need the triple-back-quote lines above and at the end